manim/mobject/coordinate_systems.py

```diff
@@ -1550,20 +1551,30 @@ def get_lines(self) -> Tuple[VGroup, VGroup]:
             self.x_axis.x_step,
             self.faded_line_ratio,
         )
+
         y_lines1, y_lines2 = self.get_lines_parallel_to_axis(
             y_axis,
             x_axis,
             self.y_axis.x_step,
             self.faded_line_ratio,
         )
+
+        # TODO this was added so that we can run tests on NumberPlane
+        # In the future these attributes will be tacked onto self.background_lines
+        # self.x_lines = x_lines1
+        # self.y_lines = y_lines1
+
+        # For some reason the above is wrong
```

Comment:
Is this supposed to be commented? 🤔 What does wrong mean here

Reply:
Hey hydrobeam, that was just when I was testing out how to set the attribute correctly. I have fixed it in the newest push.